integrations/docker/images/stage-2/chip-cirque-device-base/Dockerfile

```diff
@@ -14,7 +14,8 @@ WORKDIR /app
 RUN apt-cache policy
 
 # TODO: Use multi stage build for smaller image size.
-RUN apt-get update \
+RUN sed -i 's|http://archive.ubuntu.com/ubuntu|http://mirror.us.leaseweb.net/ubuntu|g' /etc/apt/sources.list \
```

Comment:
consider making this SED as a separate RUN clause and add a comment on why we do the change.

Reply:
done, thanks